plugins/package-managers/python/src/main/kotlin/utils/PythonInspectorExtensions.kt

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (C) 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.python.utils
+
+import java.io.File
+
+import org.ossreviewtoolkit.analyzer.PackageManager
+import org.ossreviewtoolkit.downloader.VersionControlSystem
+import org.ossreviewtoolkit.model.Hash
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.Package
+import org.ossreviewtoolkit.model.PackageReference
+import org.ossreviewtoolkit.model.Project
+import org.ossreviewtoolkit.model.RemoteArtifact
+import org.ossreviewtoolkit.model.Scope
+import org.ossreviewtoolkit.model.VcsInfo
+import org.ossreviewtoolkit.model.VcsType
+
+private const val TYPE = "PyPI"
+
+internal fun PythonInspector.Result.toOrtProject(
```

Comment:
I wonder if it's clear to future code readers that the file `PythonInspectorExtensions` is only about functions which map betweeen two models. Would it make sense to make the file name more specific?

Reply:
> Would it make sense to make the file name more specific?

I'm open for suggestions!

Reply:
Ok, so it seems to be up on me to propose names for your argument... I could call it `PythonInspectorModelUtils.kt` or `PythonInspectorModelMapper.kt` (and similar for NuGet then) if you prefer.

Reply:
> Ok, so it seems to be up on me...

No, I just wanted to point out the issue. (Because the commit message indicated to me that this separation maybe important to you) I didn't mean to ask you to change it.